SKILL.tsv

```diff
@@ -4940,13 +4940,13 @@ SKILL_20150729_004939	$Movement Speed: +#{CaptionRatio}#%{nl}Duration: #{Caption
 SKILL_20150729_004940	Back Masking Range: #{CaptionRatio}#{nl}Unable to use Back Masking nearby for 10 seconds{nl}Consumes 1 Dimensional Piece
 SKILL_20150729_004941	$Flesh Cannon
 SKILL_20150729_004942	$Flesh Hoop
-SKILL_20150729_004943	$Dig in the ground to obtain materials required for alchemy.
-SKILL_20150729_004944	$Obtain other materials by digging depending on the ground.
+SKILL_20150729_004943	$Dig up the ground to collect materials for alchemy.
```

Comment:
Dig up the ground to collect materials for alchemy. -> Dig up the ground to collect materials required for alchemy.